mobile-app/app/screens/AppNavigator/screens/Loans/screens/ConfirmEditLoanSchemeScreenV2.tsx

```diff
@@ -0,0 +1,388 @@
+              value: translate(
+                "screens/ConfirmEditLoanSchemeScreen",
+                "Prev. interest"
+              ),
+              testID: "prev_vault_interest_label",
+              themedProps: {
+                light: tailwind("text-mono-light-v2-500"),
+                dark: tailwind("text-mono-dark-v2-500"),
+              },
+            }}
+            rhs={{
+              value: new BigNumber(props.vaultInterest).toFixed(2),
+              testID: "prev_vault_interest",
+              suffix: "%",
+            }}
+          />
+        </View>
+      </ThemedViewV2>
+      <ThemedViewV2
+        dark={tailwind("border-mono-dark-v2-200")}
+        light={tailwind("border-mono-light-v2-200")}
+        style={tailwind("py-5 border-t-0.5")}
+      >
+        <NumberRowV2
+          lhs={{
+            value: translate(
+              "screens/ConfirmEditLoanSchemeScreen",
+              "New min. collateralization"
+            ),
+            testID: "new_min_col_ratio_label",
+            themedProps: {
+              light: tailwind("text-mono-light-v2-500"),
+              dark: tailwind("text-mono-dark-v2-500"),
+            },
+          }}
+          rhs={{
+            value: new BigNumber(props.newMinColRatio).toFixed(2),
+            testID: "new_min_col_ratio",
+            suffix: "%",
+          }}
+        />
+        <View style={tailwind("mt-5")}>
+          <NumberRowV2
+            lhs={{
+              value: translate(
+                "screens/ConfirmEditLoanSchemeScreen",
+                "New vault interest"
+              ),
+              testID: "new_vault_interest_label",
+              themedProps: {
+                light: tailwind("text-mono-light-v2-500"),
+                dark: tailwind("text-mono-dark-v2-500"),
+              },
+            }}
+            rhs={{
+              value: new BigNumber(props.newVaultInterest).toFixed(2),
+              testID: "new_vault_interest",
+              suffix: "%",
+            }}
+          />
+        </View>
+      </ThemedViewV2>
+      <ThemedViewV2
+        dark={tailwind("border-mono-dark-v2-200")}
+        light={tailwind("border-mono-light-v2-200")}
+        style={tailwind("py-5 border-t-0.5 border-b-0.5")}
+      >
+        <NumberRowV2
+          lhs={{
+            value: translate(
+              "screens/ConfirmEditLoanSchemeScreen",
+              "Transaction fee"
+            ),
+            themedProps: {
+              light: tailwind("text-mono-light-v2-500"),
+              dark: tailwind("text-mono-dark-v2-500"),
+            },
+            testID: "estimated_fee_label",
+          }}
+          rhs={{
+            value: new BigNumber(props.fee).toFixed(8),
+            themedProps: {
+              style: tailwind("font-normal-v2 text-sm"),
+            },
+            testID: "estimated_fee_amount",
+            suffix: " DFI",
+            usdAmount: props.fee.isNaN() ? new BigNumber(0) : props.fee,
```

Comment:
This is DFI value, Asked UX if USD value is actually needed to be displayed because   other screens are not displaying it. So, yea either we remove this or retrieve the correct usd value based from their answer

Reply:
was wondering the same thing too, i put there as placeholder wehee